test/junit/scala/collection/immutable/RangeTest.scala

```diff
@@ -1,13 +1,11 @@
 package scala.collection.immutable
 
-import org.junit.{Assert, Test}
-import org.junit.runner.RunWith
-import org.junit.runners.JUnit4
+import org.junit.jupiter.api.Assertions.{assertEquals, assertFalse, assertSame, assertTrue}
```

Comment:
this change doesn't look right. we didn't import them previously, and I don't see any changes using them